storybook/scripts/postinstall.sh

```diff
@@ -0,0 +1 @@
+#!/usr/bin/env bash
```

Comment:
This empty file was necessary, to prevent the `yarn install` command from failing. It is basically run instead of the regular postinstall script. This script does nothing, and thus does not fail at doing so.